packages/react-native/ReactAndroid/src/main/java/com/facebook/yoga/YogaAlign.java

```diff
@@ -37,8 +37,6 @@ public static YogaAlign fromInt(int value) {
       case 3: return FLEX_END;
       case 4: return STRETCH;
       case 5: return BASELINE;
-      case 6: return SPACE_BETWEEN;
-      case 7: return SPACE_AROUND;
       default: throw new IllegalArgumentException("Unknown enum value: " + value);
```

Comment:
**Suggestion:** It seems like you've removed the cases for SPACE_BETWEEN and SPACE_AROUND. If these are valid enum values, they should be handled in the switch statement.
```suggestion
      case 3: return FLEX_END;
      case 4: return STRETCH;
      case 5: return BASELINE;
      case 6: return SPACE_BETWEEN;
      case 7: return SPACE_AROUND;
      default: throw new IllegalArgumentException("Unknown enum value: " + value);
```